index.html

```diff
@@ -17,18 +17,29 @@
     <main class="container">
       <h1 class="heading-primary">Dog Breed Finder</h1>
       <section class="search-content">
-        <input
-          type="search"
-          class="search__input"
-          placeholder="Type a dog breed..."
-        />
-        <button class="search__button">
-          <svg class="search__icon">
-            <use xlink:href="sprite.svg#icon-search"></use>
+        <div class="search__input-container">
+          <input
+            type="search"
+            class="search__input"
+            placeholder="Type a dog breed..."
+          />
+          <button class="search__button">
+            <svg class="search__icon">
+              <use xlink:href="sprite.svg#icon-search"></use>
```

Comment:
Mi prosim razložiš kako to deluje?

Reply:
Na strani https://icomoon.io/ izberem svg ikone, ki jih želim za ta projekt in si jih downloadam. V project folder si shranim file symbol-defs.org, ki ga preimenujem v sprite.svg().Tukaj so vse moje svg ikone. Na ta način uporabim samo en global file. Iz tega file potem izberem ikono, ki jo potrebujem, v tem primeru je to icon-search. Zato imamo tukaj `href="sprite.svg#icon-search` s tem načinom lahko spremenim barvo svg ikone v css z fill property. To deluje samo na web serverju, zato ti na local ne pokaže ikon.

Reply:
a ni lažje da naložiš 2 ikoni v folder in da so vidne povsod?

Reply:
Razumem, da si probal kako to deluje in da poznaš več načinov, ampak v večini projektov boš imel al `font-awesome` ikonce ali pa assete k jih bojo designerji nardil in jih daš preprosto v app.

Reply:
Vrjetno je ja, to bi prišlo v poštev najbrž, če bi imel en velik projekt z veliko številom ikonami. Spremenim.

Reply:
Možno, da ga.. jaz ga sicer še nisem v produkciji in realnosti zasledu :D je pa dobr da poznaš več načinov in imaš širino. Problem trenutno je, da jaz ne morem tega vidt in reviewat.